enterprise/management/commands/seed_enterprise_devstack_data.py

```diff
@@ -254,22 +257,31 @@ def handle(self, *args, **options):
         self._create_enterprise_data_api_group()
         self._create_enterprise_enrollment_api_group()
 
+        LOGGER.info('\nCreating a new enterprise customer...')
+        site, __ = self._get_default_site()
+        enterprise_customer = self._create_enterprise_customer(site=site, name=enterprise_name)
+
         LOGGER.info('\nCreating enterprise users and assigning roles...')
         # Create one of each enterprise user type (i.e., learner, admin, operator)
         enterprise_users = [
             self._create_enterprise_user(
-                username=ENTERPRISE_LEARNER_ROLE,
+                username="{}_{}".format(ENTERPRISE_LEARNER_ROLE,slug),
                 role=ENTERPRISE_LEARNER_ROLE,
+                enterprise_customer=enterprise_customer
             ),
             self._create_enterprise_user(
-                username=ENTERPRISE_ADMIN_ROLE,
+                username="{}_{}".format(ENTERPRISE_ADMIN_ROLE,slug),
                 role=ENTERPRISE_ADMIN_ROLE,
+                enterprise_customer=enterprise_customer
+
             ),
             self._create_enterprise_user(
-                username=ENTERPRISE_OPERATOR_ROLE,
-                role=ENTERPRISE_OPERATOR_ROLE
+                username="{}_{}".format(ENTERPRISE_OPERATOR_ROLE,slug),
+                role=ENTERPRISE_OPERATOR_ROLE,
+                enterprise_customer=enterprise_customer
```

Comment:
Given:
```
def has_access_to_all_contexts(self):
        """
        Returns true if the role for this assignment is ``ENTERPRISE_OPERATOR_ROLE``,
        or if ``applies_to_all_contexts`` is true; returns false otherwise.
        """
        return (self.role.name == ENTERPRISE_OPERATOR_ROLE) or self.applies_to_all_contexts
```
I don't know if enterprise here matters?

Reply:
Good catch, will remove.